src/BinDeps.jl

```diff
@@ -297,8 +297,6 @@ macro dependent_steps(steps)
     blk
 end
 
-mypwd() = chomp(readall(`pwd`))
```

Comment:
Not used, and `readall` doesn't exist.

Reply:
Hah, that's also equivalent to the Julia function `pwd()`.